tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -2662,6 +2662,15 @@ def _GetMSBuildGlobalProperties(spec, guid, gyp_file_name):
     else:
       properties[0].append(['ApplicationType', 'Windows Store'])
 
+  msvs_windows_sdk_version = None
+  if msvs_windows_sdk_version == None and version.ShortName() == '2017':
+    vs2017_sdk = '10.0.14393.0'
```

Comment:
It may not always be installed, but is the one selected by the installer when the "Desktop development with C++" workload is selected, so it should be the most common one. If it is not installed, compilation will fail later, but it's better than failing here unconditionally. Should this be something else?